cmd/localapiservice/localapishim.go

```diff
@@ -76,10 +75,9 @@ func Java_com_tailscale_ipn_ui_localapi_LocalApiClient_doRequest(
 
 	jrespBody := jni.NewByteArray(jenv, resp)
 	respBody := jni.Value(jrespBody)
-	cookie := jni.Value(jcookie)
-	onResponse := jni.GetMethodID(jenv, shim.clientClass, "onResponse", "([BLjava/lang/String;)V")
+	onResponse := jni.GetMethodID(jenv, shim.clientClass, "onResponse", "([B)V")
 
-	jni.CallVoidMethod(jenv, jni.Object(cls), onResponse, respBody, cookie)
+	jni.CallVoidMethod(jenv, jni.Object(cls), onResponse, respBody)
```

Comment:
Note that we no longer need request ids/cookies, each request is its own instance, so Go calls back into that instances' onResponse.